src/Auth0.ManagementApi/Models/TenantSettingsBase.cs

```diff
@@ -83,13 +83,13 @@ public class TenantSettingsBase
         /// Login session lifetime, how long the session will stay valid (unit: hours).
         /// </summary>
         [JsonProperty("session_lifetime")]
-        public int? SessionLifetime { get; set; }
+        public float? SessionLifetime { get; set; }
```

Comment:
Would it be an issue to represent these as decimal values rather than as floats?

Reply:
They are IEEE 754 floats on the server side (JavaScript) so you'll only get values that can be contained in floats in .NET too.  There would be no extra precision gained using a Decimal.

Reply:
Seems reasonable to me!